tools/cleanup/README.md

```diff
@@ -1,34 +1,26 @@
 # Resources Clean Up Tool
 
-This tool is designed to clean up stale resources from gcr, for now it only deletes old images created during testing.
+This tool is designed to clean up stale resources from gcr, for now it deletes old images and clusters created during testing.
 
 ## Basic Usage
 
-Directly invoke [cleanup.sh](cleanup.sh) script with certain flags. There is no-op if invoking or sourcing this script without arguments.
+Directly invoke [cleanup.sh](cleanup.sh) script with certain flags, but don't source this script.
 
 By default the current gcloud credentials are used to delete the images. If necessary, use the flag `--service-account _key-file.json_` to specify a service account that will be performing the access to the gcr.
 
-### Clean up old images from multiple gcrs
+Projects to be cleaned up are expected to be defined in a `resources.yaml` file. To remove old images and clusters from them, call [cleanup.sh](cleanup.sh) with following flags:
 
-Projects to be cleaned up are expected to be defined in a `resources.yaml` file. To remove old images from them, call [cleanup.sh](cleanup.sh) with action "delete-old-gcr-images" and following flags:
 - "--project-resource-yaml" as path of `resources.yaml` file - Mandatory
 - "--re-project-name" for regex matching projects names - Optional, defaults to `knative-boskos-[a-zA-Z0-9]+`
-- "--days-to-keep" - Optional, default `365`
+- "--days-to-keep-images" - Optional, defaults to `365` as 1 year
+- "--hours-to-keep-clusters" - Optional, defaults to `720` as 30 days
+- "--dry-run" - Optional, performs dryrun for all gcloud functions, defaults to false
 
 Example:
 
-```./cleanup.sh "delete-old-gcr-images" --project-resource-yaml "ci/prow/boskos/resources.yaml" --days-to-keep 90```
-
-### Clean up old images from a specific gcr
-
-Cleaning up from a specific gcr is supported, except for some special ones (_knative-release_ and _knative-nightly_). Call [cleanup.sh](cleanup.sh) with action "delete-old-images-from-gcr" and following flags:
-- "--gcr-to-cleanup" as name of gcr, e.g. "gcr.io/foo" - Mandatory
-- "--days-to-keep" - Optional, default `365`
-
-Example:
-
-```./cleanup.sh "delete-old-images-from-gcr" --gcr-to-cleanup "gcr.io/foo" --days-to-keep 90```
+```./cleanup.sh "delete-old-gcr-images" --project-resource-yaml "ci/prow/boskos/resources.yaml" --days-to-keep-images 90 --days-to-keep-clusters 24```
```

Comment:
```suggestion
```./cleanup.sh --project-resource-yaml "ci/prow/boskos/resources.yaml" --days-to-keep-images 90 --days-to-keep-clusters 24```
```